pfunk/_io.py

```diff
@@ -79,6 +79,7 @@ class ResultWriter(object):
         w.write()
 
     """
+
```

Comment:
Is there a way to stop your IDE from doing this? It pollutes the commits a bit, plus I see no reason for these newlines :-)

Reply:
I could, but they are mandated by PEP257:

"Insert a blank line after all docstrings (one-line or multi-line) that document a class -- generally speaking, the class's methods are separated from each other by a single blank line, and the docstring needs to be offset from the first method by a blank line."

Reply:
Didn't know that! Cheers!

Reply:
neither did I until you asked. That is the nice thing about auto format. I can safely forget everything I know about best practices in terms of formatting my code :)